src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -100,14 +101,31 @@ type BackgroundServiceServer(state: State, client: FsacClient) =
 
     let getListOfFilesForProjectChecking file =
         match state.FileCheckOptions.TryFind file with
-        | None -> None
+        | None ->
+            if file.EndsWith ".fsx" then
+                state.Files.TryFind file |> Option.map (fun st ->
+                    async {
+                        let targetFramework = NETFrameworkInfoProvider.latestInstalledNETVersion ()
+                        let! opts = fsxBinder.GetProjectOptionsFromScriptBy(targetFramework, file, st.Lines |> String.concat "\n")
```

Comment:
`\n` -> `Environment.NewLine`?

Reply:
Doesn't matter, it's for internal usage. FCS works just fine with `\n` - we use it everywhere